pkg/apis/deployment/v1alpha/deployment_spec.go

```diff
@@ -56,7 +56,7 @@ type DeploymentSpec struct {
 	DowntimeAllowed  *bool           `json:"downtimeAllowed,omitempty"`
 	DisableIPv6      *bool           `json:"disableIPv6,omitempty"`
 
-	LocallyAttachedVolumes *bool 	`json:"locallyAttachedVolumes,omitempty"`
+	NetworkAttachedVolumes *bool     `json:"networkAttachedVolumes,omitempty"`
```

Comment:
Is this a deprecation (i.e., did we ever release a version with LocallyAttachedVolumes)

Reply:
No.